pizzeria-backend/Controllers/OrderController.cs

```diff
@@ -1,30 +1,134 @@
 ﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.SignalR;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+using pizzeria_backend.Hubs;
+using pizzeria_backend.Models.Interfaces;
+using pizzeria_backend.Services;
 
 namespace pizzeria_backend.Controllers
 {
     [Route("api/orders/v1.0")]
     [ApiController]
     public class OrderController : Controller
     {
+        private readonly IOrderService _orderService;
+        private readonly IHubContext<OrderHub> _hubContext;
 
+        public OrderController(IOrderService orderService, IHubContext<OrderHub> hubContext)
+        {
+            _orderService = orderService;
+            _hubContext = hubContext;
+        }
 
         [HttpPost()]
-        public IActionResult PostOrder()
+        public async Task<ActionResult> Order([FromBody] OrderDto Order)
+        {
+
+            var ord = await _orderService.MakeOrder(Order);
+
+            if (ord == null)
+            {
+                return BadRequest("One or more of the items are not available or in the menu");
+            }
+
+            string orderString = JsonConvert.SerializeObject(ord, new JsonSerializerSettings
```

Comment:
Are you sure you need this? You should be able to just send the order object.

Reply:
initially used it to convert from pascal case to camel case(jsons are supposed to be in camel and that caused an issue or two on the frontend). Now this part of the code is in Program.cs